drag_and_drop_v2/default_data.py

```diff
@@ -14,43 +17,62 @@
     {
       "index": 2,
       "id": "zone-2",
-      "title": _("Zone 2"),
+      "title": _("The Middle Zone"),
       "x": 86,
       "y": 210,
       "width": 340,
-      "height": 140,
+      "height": 138,
+    },
+    {
+      "index": 3,
+      "id": "zone-3",
+      "title": _("The Bottom Zone"),
+      "x": 15,
+      "y": 350,
+      "width": 485,
+      "height": 135,
     }
   ],
   "items": [
     {
-      "displayName": "1",
+      "displayName": "Goes to the top",
       "feedback": {
-        "incorrect": _("No, 1 does not belong here"),
-        "correct": _("Yes, it's a 1")
+        "incorrect": INCORRECT_FEEDBACK,
+        "correct": CORRECT_FEEDBACK.format(zone="the top")
       },
-      "zone": "Zone 1",
+      "zone": "The Top Zone",
```

Comment:
I think this needs to be translated, or else it would break for users of other languages, since "The Top Zone" would no longer match the translated zone name, e.g. "La zona de la parte superior"